### PR TITLE
Edit post

### PR DESCRIPTION
**Pull request by JatuFaique**
- Edit Post Modal
- Edit / Delete Post
- User Initials Minor fixes

### CODE REVIEW

File: src/Components/PostCard.js [hunk 1]
```diff
@@ -3,17 +3,34 @@ import { useDispatch, useSelector } from "react-redux";
...
+import EditPostModal from "./EditPostModal";
 
 function PostCard({ post }) {
   const { userDetail, token } = useSelector((state) => state.auth);
   const [comment, setComment] = useState({});
+  const [editPostModal, setEditPostModal] = useState(false);
   const dispatch = useDispatch();
 
+  const handleEditPost = (editedPost) => {
+    console.log("jj", editedPost);
```
Comment (junaidshaikh-js) [inline src/Components/PostCard.js:22]: remove console logs

File: src/styles/post.css [hunk 2]
```diff
@@ -42,3 +43,23 @@
...
+
+/* .edit_post__overlay {
+  z-index: 1000;
+  width: 100vw;
+
+  height: 111vh;
+  position: fixed;
+  transform: translate(-600px, -270px);
+  background-color: rgba(0, 0, 0, 0.5);
+} */
```
Comment (junaidshaikh-js) [inline src/styles/post.css:61]: code comments should be removed

**Review by junaidshaikh-js — APPROVED** [review]
Good to merge. Looks at the comments.

File: src/Components/PostCard.js [hunk 3]
```diff
@@ -3,17 +3,34 @@ import { useDispatch, useSelector } from "react-redux";
 import { Link } from "react-router-dom";
 import { handleBookmark, handleUnBookmark } from "../features/auth/authSlice";
```
Comment (sonishreyas) [inline src/Components/PostCard.js:4]: you can use absolute imports to make it look cleaner

File: src/features/timeline/postSlice.js [hunk 4]
```diff
@@ -19,6 +19,47 @@ export const getPosts = createAsyncThunk("posts/getPost", async () => {
   }
 });
 
+//Edit User Posts
+
+export const edituserPosts = createAsyncThunk(
+  "post/edituserPosts",
+  async (payload, { rejectWithValue }) => {
+    console.log("idhr", payload.postContent);
```
Comment (sonishreyas) [inline src/features/timeline/postSlice.js:27]: Just incase you missed it

**Review by sonishreyas — APPROVED** [review]
Code look good. I had a suggestion which you can implement later. Rest looks good.

**Pull request merged**